pyproject.toml

```diff
@@ -1,66 +1,74 @@
 [tool.poetry]
-name = "hydrolib"
-version = "0.2.0"
-description = "HYDROLIB tools for hydrodynamic and hydrological modelling workflows"
+name = "hydrolib-test"
+version = "0.6.0"
+description = "Python wrappers around D-HYDRO Suite."
```

Comment:
@RuudHurkmans : This looks like an unintended change? I guess it should stay "hydrolib", maybe with a new `version = "0.3.0"`?

Reply:
Apologies, I indeed copied from Hydrolib-core.